Test/CoreSDK.Test/Shared/TelemetryClientTest.cs

```diff
@@ -786,7 +791,70 @@ public void AllTelemetryIsSentWithDefaultSamplingRate()
 
             Assert.Equal(ItemsToGenerate, sentTelemetry.Count);
         }
-        
+
+        #endregion
+
+        #region ValidateEndpoint
+
+        [TestMethod]
+        public void SendEventToValidateEndpoint()
+        {
+            EventTelemetry telemetry1 = new EventTelemetry();
+            MetricTelemetry telemetry2 = new MetricTelemetry();
+            DependencyTelemetry telemetry3 = new DependencyTelemetry();
+            ExceptionTelemetry telemetry4 = new ExceptionTelemetry();
+            MetricTelemetry telemetry5 = new MetricTelemetry();
+            PageViewTelemetry telemetry6 = new PageViewTelemetry();
+            PerformanceCounterTelemetry telemetry7 = new PerformanceCounterTelemetry();
+            RequestTelemetry telemetry8 = new RequestTelemetry();
+            SessionStateTelemetry telemetry9 = new SessionStateTelemetry();
+            TraceTelemetry telemetry10 = new TraceTelemetry();
```

Comment:
nitpick: I would love to see clearer names than 1 through 10, even if it is just the camel case type name.